src/effects/native/balanceeffect.cpp

```diff
@@ -1,5 +1,9 @@
 #include "balanceeffect.h"
 
+namespace {
```

Comment:
If you listen to the side signal only all mono content is gone from the signal.
Since you cannot locate the low frequencies and normally part of the side signal, they are gone as well. This meas if you try to wide the stereo effect, you remove the low frequencies as well. 
A bypass for these frequencies will fix that. Remember that some setups have only a single sub woofer. This balance effect should not effect such a sub-woofer. (of cause with a parameter)